astropy/io/ascii/ui.py

```diff
@@ -593,6 +621,118 @@ def _get_guess_kwargs_list(read_kwargs):
     return guess_kwargs_list
 
 
+def _read_in_chunks(table, **kwargs):
+    """
+    For fast_reader read the ``table`` in chunks and vstack to create
+    a single table, OR return a generator of chunk tables.
+    """
+    fast_reader = kwargs['fast_reader']
+    chunk_size = fast_reader.pop('chunk_size')
+    chunk_generator = fast_reader.pop('chunk_generator', False)
+    fast_reader['parallel'] = False  # No parallel with chunks
+
+    tbl_chunks = _read_in_chunks_generator(table, chunk_size, **kwargs)
+    if chunk_generator:
+        return tbl_chunks
+
+    tbl0 = next(tbl_chunks)
+    masked = tbl0.masked
```

Comment:
Where is this line used? Seems that masked is redefined in the for look anyway?

Reply:
In the for loop it is logical-or'd with the initial value defined here.  `masked |= tbl.masked`